concordium-std-derive/src/lib.rs

```diff
@@ -193,12 +195,16 @@ fn init_worker(attr: TokenStream, item: TokenStream) -> syn::Result<TokenStream>
             #[export_name = #wasm_export_fn_name]
             pub extern "C" fn #rust_export_fn_name(#amount_ident: Amount) -> i32 {
                 use concordium_std::{trap, ExternContext, InitContextExtern, ContractState};
+                use std::convert::TryFrom;
```

Comment:
We can't use `std` directly since `concordium-std` is meant to work with `#[no_std]`.

We usually handle this by re-exporting the relevant definition from `concordium-std`. I think `convert` is already exported, so probably `concordium_std::convert::TryFrom` should work.